docs/installation.md

```diff
@@ -113,6 +113,22 @@ terminal session and verify `pipx` does run.
 > It is not recommended to install `pipx` via `pipx`. If you'd like to do this anyway, take a look at the
 > [`pipx-in-pipx`](https://github.com/mattsb42-meta/pipx-in-pipx) project and read about the limitations there.
 
+
+### On FreeBSD:
+
+- Install via package manager
+
+```sh
+pkg install -y py311-pipx
```

Comment:
Is it possible to avoid the Python version number here?

Reply:
Maybe we should to mention this in the document? Or is this common knowledge for FreeBSD users we don’t need to do it?